models/src/main/kotlin/io/provenance/api/models/cee/execute/ContractExecutionResponse.kt

```diff
@@ -1,16 +1,10 @@
 package io.provenance.api.models.cee.execute
 
-import com.fasterxml.jackson.annotation.JsonSubTypes
 import com.fasterxml.jackson.annotation.JsonTypeInfo
 import io.provenance.api.models.p8e.TxResponse
 import java.util.UUID
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
-@JsonSubTypes(
-    JsonSubTypes.Type(value = SinglePartyContractExecutionResponse::class, name = "single"),
-    JsonSubTypes.Type(value = MultipartyContractExecutionResponse::class, name = "multi"),
-    JsonSubTypes.Type(value = ContractExecutionErrorResponse::class, name = "error"),
-)
```

Comment:
Sorry, should have pointed this change out. 

Specifying subtypes is not needed as long as the `jackson-module-kotlin` is used (currently in `configureProvenance()` [here](https://github.com/provenance-io/p8e-cee-api/blob/d1504e8d8bd9fda7b02440889c9fc3a11d72468e/service/src/main/kotlin/io/provenance/api/frameworks/config/AppConfig.kt#L23)). 

From the [docs](https://github.com/FasterXML/jackson-module-kotlin/blob/29c248a4a7318c22316c2b0506171c4263ad88d2/README.md)

```
subclasses can be detected automatically for sealed classes, since all possible subclasses are known at 
compile-time to Kotlin. This makes com.fasterxml.jackson.annotation.JsonSubTypes redundant. A 
com.fasterxml.jackson.annotation.@JsonTypeInfo annotation at the base-class is still necessary.
```

Happy to add this back if you have strong opinions

Reply:
:nono: Glad it's smart enough to use sealed classes, just wasn't sure if the change was related to the fee grant stuff